apps/meteor/ee/server/startup/maxRoomsPerGuest.ts

```diff
@@ -0,0 +1,20 @@
+import { Meteor } from 'meteor/meteor';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+import type { IUser } from '@rocket.chat/core-typings';
+
+import { callbacks } from '../../../lib/callbacks';
+import { getMaxRoomsPerGuest } from '../../app/license/server/license';
+import { Subscriptions } from '../../../app/models/server';
+
+callbacks.add(
```

Comment:
I'm not sure, but I thought we were deprecating callbacks? @matheusbsilva137 Can you confirm?

Reply:
Hey Henriques! I've talked to @pierre-lehnen-rc and we're still using callbacks in this case even though they're deprecated